test/e2e/autoscaling/cluster_autoscaler_scalability.go

```diff
@@ -314,6 +323,41 @@ var _ = framework.KubeDescribe("Cluster size autoscaler scalability [Slow]", fun
 		Expect(len(nodes.Items)).Should(Equal(totalNodes))
 	})
 
+	It("should do nothing when pending pods are unschedulable [Feature:ClusterAutoscalerScalability6]", func() {
```

Comment:
I would consider renaming this test after adding the additional scale-up at the end.

Reply:
Done